packages/components/carousel/src/carousel.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Carousel as ResponsiveCarousel } from 'react-responsive-carousel';
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+import type { CarouselProps } from './carousel.types';
+import { CarouselStyled } from './carousel.styles';
+
+class Carousel extends Component<CarouselProps> {
+  render() {
+    const {
+      images,
+      showArrows = true,
+      showStatus = true,
+      showIndicators = true,
+      showThumbs = true,
+      infiniteLoop = true,
+      autoPlay = false,
+      interval = 3000,
+      ...props
+    } = this.props;
+    return (
+      <CarouselStyled>
+        <ResponsiveCarousel
+          showArrows={showArrows}
+          showStatus={showStatus}
+          showIndicators={showIndicators}
+          showThumbs={showThumbs}
+          infiniteLoop={infiniteLoop}
+          autoPlay={autoPlay}
+          interval={interval}
+          centerMode={true}
+          centerSlidePercentage={45}
+          {...props}
+        >
+          {images.map((image, index) => (
+            <div className="carousel-image-container" key={index}>
```

Comment:
import { ContainerStyled } from './carousel.styles'

```suggestion
            <ContainerStyled key={index}>
```

Reply:
Also add a proper index to better identify the element: 

```suggestion
            <div className="carousel-image-container" key={`carousel-container-${index}`}>
```